OpenRA.Game/Traits/BodyOrientation.cs

```diff
@@ -69,7 +70,7 @@ public BodyOrientation(ActorInitializer init, BodyOrientationInfo info)
 				if (info.QuantizedFacings >= 0)
 					return info.QuantizedFacings;
 
-				var qboi = self.Info.Traits.GetOrDefault<IQuantizeBodyOrientationInfo>();
+				var qboi = self.Info.Traits.WithInterface<IQuantizeBodyOrientationInfo>().FirstOrDefault();
```

Comment:
Taking the first IQBOI is a bad idea, but IIRC @reaperrr has already worked around needing to do this for sprite code.